VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,60 @@
+
+Another important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs and a reference in the sector worldwide. This partnership allows it to drive the growth of more and better tools to help engineers create and maintain asynchronous APIs while using their usual languages and working environments.
+
+AsyncAPI currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs.
+
+2. Improve the performance of the specification to make it more and more seamless. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members of around 400% over the next two years.
+
+## How did it come about?
+
+AsyncAPI started as a side project at Hitch (API Changelog) and was later founded by Fran Mendez. The AsyncAPI developers identified a lack of tools in the message-driven space. Something was missing in the documentation specifications when presented with an inversely great toolset for HTTP APIs.
+
+They began to search for a solution for standardized message-driven specification systems from this identified need. 
+
+They formulated an "API theory,” so to speak, essentially, with as many interactive API types as RESTful web services, GraphQL, and message-based architectures, AsyncAPI would serve as a common unifying language for the various formats, protocols, and specifications. All of the above results in standardized communication in the message-based system.
+
+## What AsyncAPI brings
+
+It's an extensible, protocol-agnostic communication protocol specification that helps us in any message-based case, primarily hosted in the Internet of Things (IoT), among other use cases. AsyncAPI's main contribution to the world of APIs is the standardization of communication between APIs, making messages (events) more machine-readable. This allows developers who wish to implement it to be minimized and simplified. 
+
+Having defined, documented, and accepted the communication between APIs provides us with much higher and more efficient information control. It is a multichannel tool, so it currently supports existing protocols.
+
+Therefore, the conglomerate of AsyncAPI tools allows accelerating the development of applications. Its mechanism for automating complex and essential tasks facilitates code generation, documentation generation, validators, etc. It could be said that this is a specification that facilitates, simplifies, and accelerates the work of developers working with APIs. 
+
+## How it Works / Working case study
+Case use
```

Comment:
just to clarify @alequetzalli 

when I hear someone talking about AsyncAPI use case, I'm always missing the context and always ask:

1. do you mean in general what can be done thanks to the file? like docs generation?
2. or do you mean more details, that docs generation has to be done as in case of EDA it is critical to not only share API with customers (like in case of WebSocket in crypto-trading business, or in case of MQTT in IoT business) but also to share internally in organization, to understand how and what is communicated internally (like in case of Kafka, especially in large organizations)
3. or do you mean use case + case study based on some specific company, with detailed explanation what they do?

1 - is already mentioned in one paragraph in this script
2 - I think having a dedicated video for this one would be actually nifty for this first videos block (as a result we would also get some nice `youtube shorts` and find a reason to go to TicToc 🤣 )
3 - this is super much complex, especially since we do not have these testimonials yet, long months to get there, something I would say we need to park and consider a separate block of videos about it later

@alequetzalli can you clarify which solution you propose?